samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -120,14 +120,15 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
         if (documentFile) {
             try {
                 SetDocumentImporting(true);
-                const document = await documentImportService.importDocumentAsync(
+                const message = await documentImportService.importDocumentAsync(
                     account!.homeAccountId!,
+                    (account!.name ?? account!.username) as string,
                     selectedId,
                     documentFile,
                     await AuthHelper.getSKaaSAccessToken(instance, inProgress),
                 );
 
-                handleSubmit(JSON.stringify(document), ChatMessageType.Document);
+                dispatch(updateConversation({ message, chatId: selectedId }));
```

Comment:
Do we plan to support document import from anywhere else in the future (i.e., the Files tab)? If so, I'd move this to the useChat hook in one method that handles both importDocument and updateConversation 

(or create a separate useSources hook)

Reply:
moved to `useChat`